src/crypto/tls/cfkem.go

```diff
@@ -0,0 +1,88 @@
+// Glue to add Circl's (post-quantum) hybrid KEMs to quic-go.
```

Comment:
New files should include Cloudflare copyright.

Reply:
Done